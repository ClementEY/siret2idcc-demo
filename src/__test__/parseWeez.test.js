const parseWeez = require("../parseWeez");

test("should parse weez content correctly", () => {
  expect(
    parseWeez(`MOIS,SIRET,IDCC,DATE_MAJ
2019-04,38456979400017,1505  ,2019/06/15
2019-04,82116811900013,1486  ,2019/06/15
2019-04,42370591200065,1516  ,2019/06/15
2019-04,99882350424480,2378  ,2019/06/15
2019-04,78842534600012,0993  ,2019/06/15
2019-04,43521016600060,0493  ,2019/06/15
2019-04,45170737600013,1147  ,2019/06/15
2019-04,33971149100029,1597  ,2019/06/15
2019-04,78183765300026,1518  ,2019/06/15
2019-04,51756284900011,1572  ,2019/06/15
2019-04,75326582600010,1090  ,2019/06/15
2019-04,40314218500038,1518  ,2019/06/15
2019-04,81975136300014,1996  ,2019/06/15
2019-04,50475742800017,1930  ,2019/06/15
2019-04,49200732300016,0953  ,2019/06/15`)
  ).toMatchSnapshot();
});

test("should group idcc correctly", () => {
  expect(
    parseWeez(`MOIS,SIRET,IDCC,DATE_MAJ
2019-04,AAAAA,1505  ,2019/06/15
2019-04,BBBBB,1486  ,2019/06/15
2019-04,BBBBB,1516  ,2019/06/15
2019-04,BBBBB,2378  ,2019/06/15
2019-04,AAAAA,1483  ,2019/06/15`)
  ).toMatchSnapshot();
});
