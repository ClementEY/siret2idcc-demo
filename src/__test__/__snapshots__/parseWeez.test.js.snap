// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should group idcc correctly 1`] = `
Object {
  "AAAAA": Array [
    "1505",
    "1483",
  ],
  "BBBBB": Array [
    "1486",
    "1516",
    "2378",
  ],
}
`;

exports[`should parse weez content correctly 1`] = `
Object {
  "33971149100029": Array [
    "1597",
  ],
  "38456979400017": Array [
    "1505",
  ],
  "40314218500038": Array [
    "1518",
  ],
  "42370591200065": Array [
    "1516",
  ],
  "43521016600060": Array [
    "0493",
  ],
  "45170737600013": Array [
    "1147",
  ],
  "49200732300016": Array [
    "0953",
  ],
  "50475742800017": Array [
    "1930",
  ],
  "51756284900011": Array [
    "1572",
  ],
  "75326582600010": Array [
    "1090",
  ],
  "78183765300026": Array [
    "1518",
  ],
  "78842534600012": Array [
    "0993",
  ],
  "81975136300014": Array [
    "1996",
  ],
  "82116811900013": Array [
    "1486",
  ],
  "99882350424480": Array [
    "2378",
  ],
}
`;
